### DISCUSSION TITLE
Orion JR2 BMS

### DISCUSSION THREAD

**Discussion by Gijsvm71** [Louisvdw/dbus-serialbattery#568] [New BMS requests]
I just add succesfully my DALY bms to my CCGX, THANKS for it !!!
 would it be possible to add my orion JR2 bms too ?

https://www.orionbms.com/products/orion-jr2-bms/

**Comment by Louisvdw** [comment]
The Orion JR2 does have a RS232 serial interface, but there is no reference on their website for a communications protocol that it use. If you can find that then it should be possible to add.

**Comment by Gijsvm71** [comment]
I have connected the Orion via CAN bus to my CCGX, but the information I receive is very limited, only see the battery percentage, the current and the voltage, no further information about the cell voltage. It would be nice if this could also be visible too.
